src/scancode/cli.py

```diff
@@ -131,6 +131,29 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
```

Comment:
Ah, I get it now. No the glob patterns cannot be applied to path as strings.
I'll add a note for the same in the examples.